source/pages/Home/Panel/components/Transactions/List.tsx

```diff
@@ -1,5 +1,5 @@
 import { uniqueId } from 'lodash';
-import React, { Fragment, useCallback } from 'react';
+import React, { Fragment, useCallback, useMemo } from 'react';
```

Comment:
these file changes are in another PR I don't know if we need to replicate them here.
These changes make the PR harder to review